appconf.py

```diff
@@ -93,7 +93,7 @@ def _base_config_http_common(server_name, strict_host):
 
 def base_config_https_redirect(server_name, strict_host):
     return _base_config_http_common(server_name, strict_host) + [
-        ('return', '301', f'https://{server_name}$request_uri'),
+        ('return', '301', 'https://$server_name$request_uri')
```

Comment:
How does that change anything? Primarily, I don't see how it achieves what the PR title says. What is the problem here?

Reply:
Correct me if I'm wrong, but if server_name is www.example.com example.com then this will will fail due to redirection to https://www.example.com example.com$request_uri which is invalid host name.
However with the new code it will redirect http://www.example.com/ -> https://www.example.com/ and same for example.com.